codecov.yml

```diff
@@ -1,3 +1,10 @@
+coverage:
+  status:
+    project:
+      default:
+        target: auto  # strictly increasing coverage
+        threshold: 5% # buffer for coverage drop
```

Comment:
Isn't 5% a bit high? 5% * 1890 = 94.5
3% is 56.7 which seems more apt?

(assuming consistency between runs)

Reply:
it is pretty high but left it large for flakiness